frontend/public/components/utils/button-bar.jsx

```diff
@@ -25,6 +25,7 @@ const ErrorMessage = ({ message }) => {
       className="co-alert co-alert--scrollable"
       variant="danger"
       title={t('public~An error occurred')}
+      data-test="error-message-alert"
```

Comment:
@spadgett PTAL ^^

Reply:
Hi @afreen23. Maybe I'm missing it, but I don't see where this is used in the tests. Can you point out the place?

Reply:
Ah! i got it, Initially there was a testing to check pool already existing error message, i think i added to check error message from button bar

Reply:
My bad i will remove

Reply:
Yes,  found that this was only  required herehttps://github.com/openshift/console/pull/8657#discussion_r635198910 . 
Surely we can leave this untouched. Sorry for the lame ping !  :facepalm: